src/status_im2/contexts/chat/photo_selector/events.cljs

```diff
@@ -90,7 +90,14 @@
 (rf/defn on-camera-roll-get-albums
   {:events [:on-camera-roll-get-albums]}
   [{:keys [db]} albums]
-  {:db (assoc db :camera-roll/albums albums)})
+  {:db (-> db
+           (assoc :camera-roll/albums albums)
+           (assoc :camera-roll/total-photos-count
```

Comment:
still better to move to a subscription

Reply:
sometimes its better to make this in events yes, but not in this case

Reply:
Okey

Reply:
Do you mind taking another look?